tests/test_grid.py

```diff
@@ -26,3 +27,12 @@ def test_get_grid_scrip():
     ds_test = pop_tools.get_grid('POP_gx3v7', scrip=True)
     ds_ref = xr.open_dataset(DATASETS.fetch('POP_gx3v7.nc'))
     assert ds_compare(ds_test, ds_ref, assertion='allclose', rtol=1e-14, atol=1e-14)
+
+
+@pytest.mark.parametrize('grid', pop_tools.grid_defs.keys())
+def test_HT_HU_KMU_in_grid(grid):
```

Comment:
Probably unnecessary, but I like to add comprehensive testing for any PR. I know this is at odds with the fact that other variables aren't checked. Let me know if we should retain this.